modules/issue_tracker/ajax/EditIssue.php

```diff
@@ -586,7 +586,8 @@ function getIssueFields()
         );
         $assignee_expanded = $db->pselect(
             "SELECT u.Real_name, u.UserID FROM users u
-WHERE (u.CenterID=:CenterID) OR (u.CenterID=:DCC)",
+             LEFT JOIN user_psc_rel upr ON (upr.UserID=u.ID)
```

Comment:
This is going to increase the number of rows returned.. it looks like it's fine the way it's used, but it should probably be a `SELECT DISTINCT`